internal/server/recon/recognize.go

```diff
@@ -91,8 +91,10 @@ func RecognizeEntities(
 	resp := &pb.RecognizeEntitiesResponse{
 		QueryItems: map[string]*pb.RecognizeEntitiesResponse_Items{},
 	}
+
 	// TODO: parallelize queries
 	for _, query := range in.GetQueries() {
+		query = strings.ToLower(query)
```

Comment:
Why is this needed if reconName is going to match smaller case?

Reply:
this makes things easier downstream because if there's a type, we want to match `type + " " + span` to parts in the query & capitalization of type should not matter

Reply:
Makes sense.